packages/react-components/src/components/SendBox.tsx

```diff
@@ -188,11 +204,34 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
 
   const sendTextFieldRef = React.useRef<ITextField>(null);
 
+  /* @conditional-compile-remove(file-sharing) */
+  const [showFileUploadsPendingError, setShowFileUploadsPendingError] = useState(false);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const hasIncompleteFileUploads = useCallback(() => {
```

Comment:
Why is this a `useCallback`, and not a `useMemo`? The only use of `hasIncompleteFileUploads` is on line 230, where it is called instead of being passed as a callback.

Also, I'd suggest that only the logic be conditional, and need not even be memoized (it's a boolean value that isn't that costly to compute). It can just be a file-private helper function, that's directly called where needed:

```ts

const hasIncompleteFileUploads = (props) => {
 /* @conditional-compile-remove-from-stable */
return props.activeFileUploads .....;
// Always return false on stable so that message can be sent and on error is shown
return false;
}
```

This would be the *only* conditional logic in SendBox, everything else can be unconditional, in stable build `hasIncompleteFileUploads` will always compute to false, and hence the error will never be shown.